experimental-design/README.adoc

```diff
@@ -189,23 +189,40 @@ For each modification, we will capture different properties in a `key=value` pai
 
 ..NOTE: We RECOMMEND for the modification name the UNIMOD interim name or PSI-MOD name if they are use. For custom modifications, we RECOMMEND an intuitive name.
 
-Some examples of **SDRF** with sample modifications annotated:
-
+An example of a **SDRF** with sample modifications annotated:
 
 |===
-| |comment [modification parameters] | comment [modification parameters]
+| |Comment [modification parameters] | Comment [modification parameters]
 
 |sample 1| NM=Glu->pyro-Glu; MT=fixed; PP=Anywhere; AC=Unimod:27; TA=E | NM=Oxidation; MT=Variable; TA=M
 |===
 
+[[encoding-enzymes]]
+=== Enzyme
+
+The `Comment [cleavage agent details]` property is used to capture the Enzyme information. Similar to protein modification <<encoding-protein-modifications>> we will use a key=value pair representation to encode the following properties for each enzyme:
+
+|===
+|Property |Key |Example | Required/Optional | Comment
+
+|Name of the Enzyme | NE | NM=Trypsin | Required | * Name of the Enzyme.
+
+|Database Accession| AC | AC=MS:1001251 | Optional | Accession in an external PSI-MS Ontology definition under the following category https://www.ebi.ac.uk/ols/ontologies/ms/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMS_1001045[Cleavage agent name].
+
+|Cleavage site regular expression | CS | CS=(?<=[KR]) | Optional | The cleavage site defined as a regular expression.
+|===
```

Comment:
I think this will get complex soon. We might want to keep a mapping to the common search engines enzymes and how they define the cut sites as regex to avoid the obvious ambiguities situations how to formulate the regex. Question is if we can maintain that.

Reply:
This is why the reg expression is `Optional`. I will imagine most of the users will only annotate the `NE=...`.

Reply:
The catalog with regular expressions that @ypriverol mentioned would help and I would acutally consider to make the regular expression a requirement, because e.g. `Trypsin` means different things for different engines, sometimes allowing P, sometimes not. Similarly, GluC depends on the pH. So if people just put the same name as they put into their search engine then there will be some confusion.

Reply:
I like the idea of the catalog that we can have close to this specification and maintained by all of us. I the future we can move it to OLS, PSI-MS.

Reply:
@StSchulze this can be an start following PSI-MS definitions. https://github.com/PRIDE-Archive/pride-metadata-standard/pull/18/commits/0b52f58e0b5dbe6a426cd685115ab3d5209c892f

Reply:
We could have a plain to read key value system, but the only one I come up with is complicated. But, I think you can keep it all. 

Cleavage Before CB=
Cleavage After CA=
Blocked Before BB=
Blocked After BA=

clunky.

Reply:
Agree. In a second iteration we can add more readable cleavage options. I will add an issue related with this comment.